Add-ons/UmbracoForms/Developer/AjaxForms/index.md

```diff
@@ -398,6 +400,20 @@ When posting the form, the header value generated can be provided, where it will
     });
 ```
 
+### API Key
+
+The antiforgery token security approach is valid for when building a client-side integration with API calls made from the browser.
+
+Providing the token isn't possible though in other headless situations such as server-to-server requests. In these situations, an alternative approach to securing the API is available.
+
+Firstly, with server-to-server integrations you will want to disable the antiforgery token protection.
+
+This is done by setting the `Umbraco:Forms:Security:EnableAntiForgeryTokenForFormsApi` configuration key to a value of `false`.
+
+You should then configure an API key `Umbraco:Forms:Security:FormsApiKey`.  This can be any string value, but shouldn't be easily guessable by a brute force attack.
```

Comment:
```suggestion
You should then configure an API key `Umbraco:Forms:Security:FormsApiKey`.  This can be any string value but shouldn't be guessable by a brute force attack.
```

Reply:
I saw this suggestion, but actually think this is fine here.  Strictly speaking any key is "guessable" (given enough time) so "easily guessable" is more correct here.